pkg/config/nerdctl_config_applier_test.go

```diff
@@ -231,10 +238,15 @@ func TestNerdctlConfigApplier_Apply(t *testing.T) {
 
 			ctrl := gomock.NewController(t)
 			fs := afero.NewMemMapFs()
+			fc := &Finch{
+				Memory:  pointer.String("2GiB"),
+				CPUs:    pointer.Int(4),
+				VMType:  pointer.String("qemu"),
+				Rosetta: pointer.Bool(false),
+			}
 			d := mocks.NewDialer(ctrl)
-
 			tc.mockSvc(t, fs, d)
-			got := NewNerdctlApplier(d, fs, tc.path, tc.hostUser).Apply(tc.remoteAddr)
+			got := NewNerdctlApplier(fc, d, fs, tc.path, tc.hostUser).Apply(tc.remoteAddr)
 
```

Comment:
Need a test case that includes host-gateway-ip in finch config. You can run `make coverage` to self-check if any lines are not covered.

Reply:
Oh ok I see